.gitlab-ci.yml

```diff
@@ -1510,7 +1510,7 @@ build_mender-artifact:
       - $RUN_INTEGRATION_TESTS == "true"
   image: docker
```

Comment:
```suggestion
  image: docker:18.08
```

Reply:
These images also need to change as they will have `DOCKER_HOST` pointing to the ssl port on the docker host.